spec/v0-1.md

```diff
@@ -68,11 +68,14 @@ The expression `(atom <e>)` evaluates to `t` if `<e>` evaluates to an atom (non-
 
 ### `cons`, `car`, `cdr`
 
-The expression `(cons <a> <d>)` produces a pair whose `car` is `<a>` and whose `cdr` is `<d>`. The expression `(car <e>)` and `(cdr <e>)` return the `car` and `cdr` of the pair resulting from the evaluation of `<e>`.
-It is an error to call `car` or `cdr` on an argument value that is not a pair.
+The expression `(cons <a> <d>)` produces a pair whose `car` is the result of evaluating `<a>` and whose `cdr` is the result of evaluating `<d>`. The expression `(car <e>)` and `(cdr <e>)` return the `car` and `cdr` of the pair resulting from the evaluation of `<e>`.
+
+In the special case where `<a>` evaluates to a character and `<d>` evaluates to a string, `(cons <a> <d>)` produces a string rather than a pair. When `<e>` is a string rather than a pair, the expression `(car <e>)` returns the first character (or `nil` for an empty string) and the expression `(cdr <e>)` returns the substring without the first character (or the empty string for an empty string).
```

Comment:
The parentheticals are slightly ambiguous:
- '`nil` for an empty string' means when `<a>` is the empty string:  `(car "")` => `nil`
- 'or the empty string for an empty string' includes the case where `<e>` is *not* the empty string: `(cdr "a")` => `""`.